src/applications/personalization/profile/tests/notification-settings.unit.spec.js

```diff
@@ -0,0 +1,37 @@
+import { expect } from 'chai';

```

Comment:
can you move the test for this selector into the existing unit tests for the selectors: `src/applications/personalization/profile/tests/selectors.unit.spec.js`